### ISSUE TITLE
[Suggestion] Portable Peripherals

### ISSUE THREAD

**Issue by masonpinder** [OpenMods/OpenPeripheral#235]
Can you make the scanner become a portable peripheral, like make it connect to the peripheral glasses? I would find that extremely useful, being able to connect a peripheral up to a player. Sorry if this is impossible, But just an idea. I think that it would be extremely helpful though, and I couldn't think of any other way to make the scanner become portable.

Another idea I had was to possibly make some integration for a recipe that takes (Drive + Ender chest + Floppy disk + (Modem/wireless modem?)) And you can get an ender modem, which has an infinite range. I have used drives and ender chests to send data from one dimension to another, and find that making a modem that works infinite range would help it a LOT, and just make it easier for other people to use. Thanks!!!

**Comment by Selim042** [comment]
I think a noteblock attachment for the glasses would be cool too, playing notes to the player wearing the glasses only.

**Comment by masonpinder** [comment]
I agree with the idea of noteblocks! Maybe they could make the player interface become something with different addons, like a turtle equipping a sword and pick or axe, making it so that the glasses can have peripherals on it, possibly connected to the player interface block? Food module, sound module, inventory module (so it can move things in your inventory and push your inventory into the world possibly like pressing Q, or by spitting it into nearby interfaces?), and the scanner module added to it so it's mobile. Then it would be extremely useful for my adventure maps I want to make! Also, could be extremely overpowered or just good things added into computercraft.

**Comment by nevercast** [comment]
Future note: Scanner is part of OpenPeripheral Addons.
